mula/scheduler/schedulers/boefje.py

```diff
@@ -997,3 +997,19 @@ def get_boefjes_for_ooi(self, ooi) -> list[Plugin]:
         )
 
         return boefjes
+
+    def calculate_deadline(self, task: Task) -> datetime:
+        """Calculate the deadline for a task.
+
+        Args:
+            task: The task to calculate the deadline for.
+
+        Returns:
+            The calculated deadline.
+        """
+        # Does the boefje have an interval defined?
+        interval = utils.deep_get(task.data, "boefje", "interval")
+        if interval is not None:
+            return datetime.now(timezone.utc) + timedelta(seconds=interval)  # FIXME: check if it is seconds or minutes
```

Comment:
The expected interval in the boefje schema is in minutes, as that is fine gained enough for our monitoring purposes.